staker-rewards/data/cache-transactions.json

```diff
@@ -0,0 +1,43 @@
+{
+  "transactions": [
+    {
+      "hash": "0xc00626a9541466079ea12662ad5356311cbccb2a478a11fcb01c6bc769fc4aa5",
+      "from": "0x13aba80a5bf56720ec44e26c06e88a21f4d0a6d9",
+      "to": "0x27321f84704a599ab740281e285cc4463d89a3d5",
+      "block_number": 11092262,
+      "method": "deauthorizeSortitionPoolContract",
+      "decoded_input": [
```

Comment:
If we decide to add another API we can store plain data in the cache and decode it later when we use it.

Reply:
Having a cache per implementation is another option though it could be risky given that we have multiple sources of truth in this case. I'd rather consider implementing a decoder that would turn API-specific data format into a common one, that would land in a cache. Another benefit of this approach is that even if one API gets disabled, you can pump the cache from another API, starting from the moment the previously used API failed on. Not worth wasting time on this discussion though, the current solution is 👌 for what we currently need.